Source/player.h

```diff
@@ -160,8 +160,14 @@ struct PlayerStruct {
 	Sint32 _py;      // Tile Y-position of player
 	Sint32 _pfutx;   // Future tile X-position of player. Set at start of walking animation
 	Sint32 _pfuty;   // Future tile Y-position of player. Set at start of walking animation
-	Sint32 _ptargx;  // Target tile X-position for player movment. Set during pathfinding
-	Sint32 _ptargy;  // Target tile Y-position for player movment. Set during pathfinding
+
+	/*
+	* These values store the target position for non-local players during network play.
+	* This is needed because player positions drift over time due to desyncs, so we have
+	* clients try to pathfind non-local players to this position.
+	*/
+	Sint32 _ptargx, _ptargy;

```

Comment:
I don't think this is going to jive with clang-tidy, though it makes good sense. Maybe we should instead use a point type, that would greatly simplify all the structures as well.

Reply:
Should I just change it back to this?

```
Sint32 _ptargx;
Sint32 _ptargy;
```

Reply:
No, lets try it out and see what happens once I get to this file :)